.github/workflows/flatpak-build.yml

```diff
@@ -1,7 +1,7 @@
 name: Flatpak Build
 on: [push, pull_request]
 jobs:
-  build:
+  flatpack_build:
```

Comment:
```suggestion
  flatpak_build:
```

Reply:
Hah! My knowledge of spelling bit me 😜

Thanks!